### PR TITLE
Devfest section is added. 

### LINKED ISSUE #427: DevFest 2015 season
DevFest 2015 season is beginning now. Do you all think that we should include it from now.

### PR DESCRIPTION
**Pull request by tasomaniac**
Fixes #427

### CODE REVIEW

**Comment by tasomaniac** [comment]
Here are the screenshots: (Study Jams tag is used to be able to fill the list temporarily. ) :smile: 

We may change the colors. Sugeestions are welcome. 

shamulyz28etasomaniac07282015172833
shamulyz28etasomaniac07282015172854
shamulyz28etasomaniac07282015172902

**Comment by PareshMayani** [comment]
Looks perfect to me. Let me know if I should merge it!

File: app/src/main/java/org/gdg/frisbee/android/Const.java [hunk 1]
```diff
@@ -39,20 +39,21 @@ private Const() {
     public static final String EXTRA_TAGGED_EVENT = "org.gdg.frisbee.TAGGED_EVENT";
     public static final String EXTRA_TAGGED_EVENT_CACHEKEY = "org.gdg.frisbee.TAGGED_EVENT_CACHEKEY";
     //Special Events Dates
-    public static final DateTime START_TIME_IOEXTENDED =    new DateTime(2015, 5, 1, 0, 0);
-    public static final DateTime START_TIME_STUDY_JAMS =    new DateTime(2015, 2, 1, 0, 0);
+    public static final DateTime START_TIME_DEVFEST =       new DateTime(2015, 9, 1, 0, 0);
```
Comment (PareshMayani) [inline app/src/main/java/org/gdg/frisbee/android/Const.java]: I guess it is displaying study jam events due to the starting date of DevFest, is it true?

**Comment by tasomaniac** [comment]
Nooo :) 
I've just flick the tag for devfest to fill the list for screenshots. When you give devfest as the tag, nothing is shown currently.

**Comment by tasomaniac** [comment]
Feel free to merge it since it doesn't change functionality, just add a special event.

**Comment by PareshMayani** [comment]
@tasomaniac ohh then it's perfect.

**Pull request merged**